packages/prisma/schema.prisma

```diff
@@ -9,7 +9,7 @@ datasource db {
 
 generator client {
   provider        = "prisma-client-js"
-  previewFeatures = ["views"]
+  previewFeatures = ["views", "metrics"]
```

Comment:
Should we put this behind an env variable and have it turned off by default? Might affect perf

Reply:
I'm not too sure if we can do that for enabling preview features.